teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
Are you saying that anyone using this library will have the same values? If that's not true, it has to be configurable. Or determined by looking up statuses and types by name.